cmd/snap-bootstrap/triggerwatch/evdev.go

```diff
@@ -205,6 +205,27 @@ func (e *evdevKeyboardInputDevice) Close() {
 
 type evdevInput struct{}
 
+func (e *evdevInput) Open(filter triggerEventFilter, node string) (triggerDevice, error) {
+	evdevDevice, err := evdev.Open(node)
+	if err != nil {
+		return nil, err
+	}
+	kc, ok := strToKey[filter.Key]
```

Comment:
nitpick: please pick a longer name for `kc`; I guess it stands for "key code", but I could also be wrong :-)